docs/04-guides/on-demand/01-upload-video-asset/02-api.mdx

```diff
@@ -16,6 +16,8 @@ The Livepeer API allows you to send video files to Livepeer and get them ready
 for optimized playback. Videos can be provided either by you (static content) or
 your users, given your application offers an interface for them to do so.
 
+You can [upload via URL](#uploading-via-url), [upload locally](#uploading-locally) or by [resumable upload](#resumable-upload).
```

Comment:
"You can upload a video asset via a url or locally via a [direct upload](#direct-upload) or a [resumable upload](#resumable-upload)."